src/Models/HydrostaticFreeSurfaceModels/compute_w_from_continuity.jl

```diff
@@ -18,12 +19,38 @@ compute_w_from_continuity!(model; kwargs...) =
 compute_w_from_continuity!(velocities, arch, grid; parameters = w_kernel_parameters(grid)) = 
     launch!(arch, grid, parameters, _compute_w_from_continuity!, velocities, grid)
 
+######################################################
+# The derivative of the moving grid is:
```

Comment:
This comment says something about a moving grid, but I think the code here is general and also works for static grids. This comment is very confusing, it will make people think that we always have a moving grid